### ISSUE TITLE
LIP-73: migrate() w/ msg.sender auth in L1 Migrator

### ISSUE THREAD

**Issue by yondonfu** [livepeer/protocol#486]
See the `migrate()` description for the L1 Migrator in LIP-73. The implementation for this issue should assume msg.sender authorization. Signature authorization is out of scope of this issue.

The implementation of the L1LPTGateway call at the end of this function is currently blocked by livepeer/arbitrum-lpt-bridge#20. So, for now, let's add a comment at the end of the function noting that there should be a call to the L1LPTGateway that can be filled in after livepeer/arbitrum-lpt-bridge#20 is complete.

Note: The method signature for `migrate()` will eventually need to include the gas fields required for the L2 tx, but this is out of scope for this issue for now until the previous arbitrum-lpt-bridge issue is addressed and the method signatures there are finalized.

**Comment by yondonfu** [comment]
Closed by livepeer/arbitrum-lpt-bridge#24
